README.md

```diff
@@ -20,27 +20,81 @@ This repository has been created to coordinate developers' work in comparing Ope
 
 ## Development machine setup
 
-## Running an IIIF Image Server (Mac)
+This is a three-step process:
 
-This directory contains the Java files for a Cantaloupe Image Server and an `images` directory from which local images are served. 
+1. Run the image server:
+	* [On Unix systems](##running-an-iiif-image-server-unix)
+	* [On Unix systems](##running-an-iiif-image-server-windows)
+2. [Running the web server](##running-the-web-server)
```

Comment:
See below: if we're running the web server only for the purpose of UV, then we can probably make this a two-step process.